<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchForCarsWindow</class>
 <widget class="QDialog" name="SearchForCarsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск автомобилей в каталоге</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 240, 239);
font: 600 9pt &quot;Montserrat&quot;;
color: rgb(55, 51, 53);</string>
  </property>
  <widget class="QLabel" name="topLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 10pt;
qproperty-alignment: 'AlignCenter';</string>
   </property>
   <property name="text">
    <string>Выберите параметры для поиска и введите значения</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="yearCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabelFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>От:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEditFrom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabelTo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mileageLabelFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>От:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mileageCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Пробег(км)</string>
   </property>
  </widget>
  <widget class="QLabel" name="mileageLabelTo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mileageLineEditTo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mileageLineEditFrom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabelFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>От:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="priceCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Стоимость $</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabelTo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEditTo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceLineEditFrom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="fuelLabelFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>От:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fuelCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Объем топ-
ливного бака</string>
   </property>
  </widget>
  <widget class="QLabel" name="fuelLabelTo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fuelLineEditTo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fuelLineEditFrom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEditTo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabelTo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>До:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batteryLineEditFrom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batteryLineEditTo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabelFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>От:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="batteryCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Емкость
аккумулятора</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="brandCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Марка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="modelCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Модель</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 51, 53);
font: 600 9pt &quot;Montserrat&quot;;
border-style: solid;
border-width: 0px;
padding: 5px;
background-color: rgb(200, 215, 210);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hybridTypeCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator {
    border: solid;
	border-color: #657A84;
	border-width: 1px;	
}
QCheckBox::indicator:checked {
	border: none;
	background-color: rgb(215, 67, 57);
}</string>
   </property>
   <property name="text">
    <string>Тип гибрида</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hybridTypeComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>410</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox 
{
	color: rgb(30, 36, 39);
	font: 600 9pt &quot;Montserrat&quot;;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px
}
QComboBox::drop-down
{
	border:none;
}
QComboBox::down-arrow
{
	image: url(:/rcs/free-icon-down-arrow-2985150.png);
	width:10px;
	height:10px;
	margin-right: 7px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>1. Последовательный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2. Параллельный</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>331</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
background-color: rgb(244, 240, 239);
border: none;
border-radius: 15px;
font: 600 9pt &quot;Montserrat&quot;;
}
QListWidget::item:selected 
{
	background-color: #E0E8E6; 
	border-radius: 12px; 
	padding: 2px; 
	border: none 
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #9AB6AC;
}
QPushButton{
	color: rgb(55, 51, 53);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 10px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(200, 215, 210);
	padding: 5px;
}

</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchPushButton">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>520</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Найти автомобили</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt;</string>
   </property>
   <property name="text">
    <string>Результаты поиска</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #CC3329;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="savePushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover
{
	background-color: #CC3329;
}
QPushButton{
	color: rgb(244, 240, 239);
	font: 600 10pt &quot;Montserrat&quot;;
	border-radius: 12px;
	border-style: solid;
	border-width: 0px;
	background-color: rgb(215, 67, 57);
	padding: 5px;
}</string>
   </property>
   <property name="text">
    <string>Сохранить результаты в файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoAboutFileLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>560</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #4AAE58;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
